query/src/main/java/org/infinispan/query/impl/SecurityActions.java

```diff
@@ -29,6 +29,10 @@ private SecurityActions() {
             AccessController.doPrivileged(action) : Security.doPrivileged(action);
    }
 
+   static String getSystemProperty(String propertyName) {
```

Comment:
Won't be better to return Optional<String> ? and then deal with it in the code. Just suggesting

Reply:
I hate Optional. Sorry :)

Reply:
there such good tings to do with Optional, specially since java 9 even moreee! but ok 😄